README.md

```diff
@@ -62,9 +62,33 @@ of `sdk.Record.Key` field are taken from `sdk.Payload.After` by the keys of this
 
 For each record, the connector adds a `redshift.table` property to the metadata that contains the table name.
 
+## Destination
+
+The Redshift Destination Connector allows you to move data to a Redshift table with the specified `dsn` and `table`
+configuration parameters. It takes an `sdk.Record` and parses it into a valid SQL query.
+
+### Configuration Options
+
+| name         | description                                                                                                                                                           | required | example                                               |
+|--------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|-------------------------------------------------------|
+| `dsn`        | [DSN](https://en.wikipedia.org/wiki/Data_source_name) to connect to Redshift.                                                                                         | **true** | `postgres://username:password@endpoint:5439/database` |
+| `table`      | Name of the table the connector must read from.                                                                                                                       | **true** | `table_name`                                          |
+| `keyColumns` | Comma-separated list of column names to build the where clause in case if `sdk.Record.Key` is empty. See more: [Destination Key handling](#destination-key-handling). | false    | `id,name`                                             |
+
+#### Destination Key Handling
```

Comment:
And since this section is a child section of the "_Destination_", isn't it enough to call it "_Key Handling_"? The same goes for the Source.

So the table of contents will look like this:
> - Destination
>   - Configuration Options
>   - Key Handling
>   - Table Name

Instead of this one:
> - Destination
>   - Configuration Options
>   - Destination Key Handling
>   - Table Name

Reply:
I did it due to the links to these paragraphs.
But, yeah, probably I should rename it.

Reply:
You could declare links like so: `[Key Handling](#key-handling)` and `[Key Handling](#key-handling-1)` to point to different sections with the same name. But, yeah, this makes links less readable.

Reply:
Deal👍